CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 You can find and compare releases at the GitHub release page.
 
+
+### Fixed
+- Add more ReturnTypeWillChange for PHP 8.1 compatibility
+
```

Comment:
@Messhias @mfn You should remove this before or after you merge if your happy to continue with https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/107

Reply:
The #107 it's just populated the release story branch and not CHANGELOG.md file.

Reply:
Sorry, I don't understand? #107 will automatically update the CHANGELOG when you create a release so manually adding changes here is no longer needed.

Reply:
> Sorry, I don't understand? #107 will automatically update the CHANGELOG when you create a release so manually adding changes here is no longer needed. See video for what you need to do when you create a release: [#107 (comment)](https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/107#issuecomment-1018400549)
> 
> The text content that is generated will automatically end up in the CHANGELOG on publish.

In my case here at work, I prefer using the automated generated changelog on releases and I use a post-commit hook to have tracking.

Reply:
I have discussed this with different maintainers all day 🤣 
You guys can decide between you whether to stick with it or not 😄. 

I would say do the next release with it in see what you think. If you don't like just use the revert button on #107 ❤️

Reply:
> I would say create the next release with it in and see what you think.

Sounds good to me 😄